lab-app/src/App.js

```diff
@@ -0,0 +1,49 @@
+import './App.css';
+import ironhackLogo from './images/ironhack-logo-xs.png'
+import menuTop from './images/menu-top-xs.png'
+import icon1 from './images/icon1.png'
```

Comment:
Try to use descriptive names so it's easier to maintain in the future application.

```suggestion
import declarativeIcon from './images/declarative.png'
```